gui/projectfile.ui

```diff
@@ -6,7 +6,7 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>940</width>
+    <width>947</width>
```

Comment:
I would prefer that this is reverted. Qt Creator often resize windows like this even though you didn't do anything :-(